src/FilesSidebarTabApp.vue

```diff
@@ -19,26 +19,294 @@
   - along with this program. If not, see <http://www.gnu.org/licenses/>.
 -->
 <template>
-	<p>Talk tab coming soon</p>
+	<div class="talkChatTab">
+		<div v-if="fileInfo && fileInfo.isDirectory()" class="emptycontent">
+			<div class="icon icon-talk" />
+			<h2>Conversations are not available for folders</h2>
+		</div>
+		<div v-else-if="isTalkSidebarSupportedForFile === undefined" class="emptycontent ui-not-ready-placeholder">
```

Comment:
`hasChatTab` sounds like "_show the chat tab or not_", but this is "_the chat tab is shown because the file could have a conversation, but are the preconditions to be able to create a conversation fulfilled?_", which is different. I agree, though, that `isTalkSidebarSupportedForFile` may not be very clear either... I just copied it from Talk 7 ;-)